src/ts/components/seekbar.ts

```diff
@@ -765,18 +769,19 @@ export class SeekBar extends Component<SeekBarConfig> {
     if (this.config.vertical) {
       px = this.seekBar.height() - px - this.seekBarPlaybackPositionMarker.height();
     }
+    const propertyValueString = 'calc(' + SeekBar.TRANSLATE_OFFSET + '% + ' + px + 'px)';
```

Comment:
`calc` won't work in old Android WebViews so we shouldn't use it. Is this really necessary? Why can't we calculate in JS like before?

Reply:
It's possible in JS but i thought the support for `calc` is good enough. 

I saw that calc is supported since > 4.4.4 but 4.4 and 4.4.4 has < 1% and our `autoprefixer` setting is >1% so i assumed that it is ok to use calc.

Reply:
Sounds reasonable, ok then let's use `calc` and see what happens :) https://caniuse.com/#search=calc says that 4.4 and 4.4.4 partially support `calc` and are only missing the ability to multiply and divide... since we're only adding here, this might even work.

Reply:
Actually, if you revert the `transform` to `left` in `_seekbar.scss` this becomes unnecessary and can be removed :)